cvxpy/lin_ops/canon_backend.py

```diff
@@ -2258,13 +2263,16 @@ def get_tensor_representation(self, row_offset: int) -> TensorRepresentation:
             for parameter_id, parameter_matrix in variable_tensor.items():
                 p = self.param_to_size[parameter_id]
                 m = parameter_matrix.shape[0] // p
+                coo_time = time.time()
```

Comment:
This is how I noticed the difference between the two backends. 
I saw that the GB took more than 2 seconds while SciPy only took 0.1 second, which was really concerning.